app/pug/user_profile.pug

```diff
@@ -52,12 +50,11 @@ block content
       
       dt Roles
       dd
-        pre #{JSON.stringify(user.roles)}
-    
+        code #{JSON.stringify(user.roles)}
+      
       dt Permissions granted by these Roles:
       dd
-        <pre>#{JSON.stringify(user.permissions)}</pre>
-    
+        code #{JSON.stringify(user.permissions)}
```

Comment:
Semantic nitpick:  `code` is for displaying computer code.  If the idea is to just use a monospace font, use CSS.  If the semantic is preformatted text, `pre` is the correct element to use.  See here:  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code

Reply:
Noted, thanks.  Will change it to use CSS and add to the PR.